scripts/cfme/create_rhev_provider.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# See following link for more info:
+# http://manageiq.org/documentation/development/rest_api/models/providers/
+
+export CFME_IP="192.168.252.16"
+export RHEV_IP="192.168.252.13"
+export RHEV_ADMIN_NAME="admin@internal"
```

Comment:
Do you really need to do export on these? I think it would suffice to simply do:
```CFME_IP="192.168.252.16"``` etc.